### ISSUE TITLE
Auto-hide menubar.

### ISSUE THREAD

**Issue by 5310** [zeit/hyperterm#158]
The menubar looks out of place with the app's design on desktop environments that show it, it doesn't perform any important functions either.

I propose to hide it by default with the [`autoHideMenuBar`](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions) option in Electron, making it still accessible with 'alt' as in most such hidden menus.

This could probably be done with an addon, but it feels like something the app would be doing by default and is missing purely because cross-platform polish hasn't started yet? At least that's why I believe this change should be made on the base app.

**Comment by leo** [comment]
@5310 Which OS are you on?

**Comment by 5310** [comment]
I am on Xubuntu 14.04 (running XFCE) and Windows 10.

**Comment by avindra** [comment]
I'm running openSUSE Tumbleweed, would love to see this configuration option. Looks quite ugly with a tiling window manager (i3wm)

**Comment by esphen** [comment]
I completely agree with @avindra. I am also on a tiled window manager, and it looks terrible. See this screenshot for just how out of place it looks (`HyperTerm` is in the bottom right,  the two other windows are `urxvt`)

![2016-07-17-134730_swaygrab](https://cloud.githubusercontent.com/assets/5845924/16900466/90c50f6e-4c25-11e6-957d-764fd39eafe2.png)

**Comment by rumpelsepp** [comment]
the same here with i3.

**Comment by tomru** [comment]
feel free to use this _hypercomplex_ plugin https://www.npmjs.com/package/hyperterm-hidemenu :joy_cat:

**Comment by iamstarkov** [comment]
@tomru neat, can i ask you to add screenshot to readme?

**Comment by CodeTheory** [comment]
This should be in 1.0.0!
